RR.App.Tests/Controllers/HRIS/EmployeeDocumentControllerUnitTest.cs

```diff
@@ -172,23 +161,8 @@ public async Task UpdateEmployeeDocumentReturnsOkResult()
     [Fact]
     public async Task UpdateEmployeeDocumentReturnsNotFoundResultWhenExceptionThrown()
     {
-
         EmployeeDocumentDto broken = new EmployeeDocumentDto
         {
-            Id = 1,
```

Comment:
Why is this code removed? One must be able to assert if the values are being updated. And if this variable is not being used in this test, remove it entirely